### PR TITLE
Add mnemonic device to C# ?: operator docs

### LINKED ISSUE #11802: C# Language Reference: Add mnemonic device to ?: operator to help retention of how evaluation works
**Help us make content visible**
- Tell us what search terms you used and how you searched docs.
N/A

- Tell us what docs you found that didn't address your concern.
N/A

**Describe the new topic**
- Abstract:
I've always had trouble remembering how the ?: operator evaluates off the top of my head, and I've come up with a mnemonic device to help me remember this. I believe that this mnemonic device could be helpful to others. The mnemonic device I use is `is this cond true ? yes : no`, with the ? part acting as a question mark for the previous statement.

- Suggest a location in the Table of Contents
This change would go directly into [the existing page](https://github.com/dotnet/docs/blob/master/docs/csharp/language-reference/operators/conditional-operator.md) for the conditional operator.

A mnemonic device is defined as: any learning technique that aids information retention or retrieval in the human memory. Mnemonics make use of elaborative encoding, retrieval cues, and imagery as specific tools to encode any given information in a way that allows for efficient storage and retrieval.

I think that it would be helpful to others to add this mnemonic device to the documentation for this operator. I would be happy to work on this, and work on refining the language of the device as it is quite informal at the moment.

### PR DESCRIPTION
**Pull request by akmadian**
## Summary

Added a mnemonic device to help retention of how the C# ?: operator evaluates. The language could use some refining as it is quite informal, but I figured it would be best to get input from others on this instead of trying to struggle through it myself.

Fixes #11802, Here's the PR @BillWagner

### CODE REVIEW

File: docs/csharp/language-reference/operators/conditional-operator.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ is evaluated as
 a ? b : (c ? d : e)
 ```
 
+A handy [mnemonic device](https://www.google.com/search?q=mnemonic+device&rlz=1C1CHBF_enUS811US811&oq=mnemonic+device&aqs=chrome..69i57j69i61j0l4.2439j0j7&sourceid=chrome&ie=UTF-8) you can use to remember how this operator evaluates is by asking: `is this cond true ? yes : no`, with the ? part of the operator acting as a question mark for the previous statement, and the consequent acting as the logical response to this question.
```
Comment (Thraka) [inline docs/csharp/language-reference/operators/conditional-operator.md]:
```suggestion
A handy mnemonic device you can use to remember how this operator evaluates is by asking: `is this cond true ? yes : no`, with the ? part of the operator acting as a question mark for the previous statement, and the consequent acting as the logical response to this question.
```

We wouldn't want to put in search queries directly into the documentation.

**Review by Thraka — CHANGES_REQUESTED** [review]
@akmadian This looks good! We don't want to have any search queries embedded in the text though. Someone can easily just do their own search if they don't know what a mnemonic device is.

Also, we shouldn't have the mnemonic device in a `code block` as the code blocks aren't translated to other languages. I would use something like.

```markdown
A handy mnemonic device you can use to remember how this operator evaluates is by asking: 

> Is this condition true ? yes : no

The `?` part of the operator acting as a question mark for the previous statement, and the consequent acting as the logical response to this question.
```

**Comment by akmadian** [comment]
All makes good sense, will make changes when I get back home :)

**Comment by BillWagner** [comment]
@akmadian It looks like you accepted @Thraka 's suggestion. (Thanks for reviewing while I was OOF).

If you're done, let me know and I'll :shipit:.

**Comment by Thraka** [comment]
@akmadian The wording is still off though because of the code block. I think you should also spell out condition.

CC @BillWagner

**Comment by akmadian** [comment]
@Thraka Code block has been added, and I've spelled out conditional. Great suggestions, thanks!

Let's :shipit:!

**Comment by BillWagner** [comment]
I like it! @Thraka do you want to approve it?

**Review by Thraka — APPROVED** [review]

**Comment by Thraka** [comment]
@akmadian Thanks again 😄

**Comment by akmadian** [comment]
Happy to help!

**Pull request merged**